.github/workflows/packaging.yaml

```diff
@@ -0,0 +1,219 @@
+name: Package linux binaries
+
+env:
+  VERSIONS_LIST: "0.10.7,0.13.6,0.14.1,0.17.3,0.18.3,0.19.4,0.21.1,0.24.0,0.26.5,0.28.0,0.29.1,0.31.2,0.31.3,0.32.0,0.32.2,0.33.0"
```

Comment:
This list needs to be updated upon every release, we can automate this. One way to achieve this is to trigger build on new release tag.